src/components/PrintModal/WatermarkModal/WatermarkModal.js

```diff
@@ -276,6 +270,33 @@ class WatermarkModal extends React.PureComponent {
     return locationSetting.isSelected;
   });
 
+  isTextValid = text => text !== undefined && text !== null && text !== '';
```

Comment:
Could you tell me in what situations will text be `undefined` or `null`? It looks like the default value is `''`...

Is it possible that we just do a falsy check? like
```javascript
if (!currLocationSetting[FORM_FIELD_KEYS.text]) {
  ...
}
```

Reply:
correct. the default initial value is always an empty string. so i don't think it can be `undefined` or `null`. i guess i was thinking and trying too hard to attempt to make things robust.

Seeing my terenary operator from my function i have created. yes. i think the falsey check will suffice.